src/data/dummyData.ts

```diff
@@ -116,7 +108,7 @@ export const DUMMY_DATA_QUESTIONS: Question[] = [
         "All changes have already been added and staged, now we need to commit and push our changes onto main.",
         "Use git commit commands in the terminal to complete this task and click submit to check!"
       ],
-      desiredGraphImgUrl: "ActivityCommitGraph.png"
```

Comment:
did you test if it works? Cause idk do you need to specify the path.

Reply:
It isn't implemented yet, and that was already like that, so I modelled all the other URLs like that

Reply:
You don't need to specify the path, if you put the images in the Public folder, instead of assert

Reply:
yeah currently its in the assets folder (see #139), but i can move it to public if u want, its a very minor thing tho